### PR TITLE
add doc links

### PR DESCRIPTION
**Pull request by joemcelroy**
- Added homepage url to each NPM package so link appears on NPM
- updated all the markdown links to the relevant search-ui doc page

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +33,10 @@ search experience up and running in minutes.
 
 <img src="docs/search-ui/images/screenshot.png" width="600">
 
+### Docs
+
+Checkout our [documentation site](https://docs.elastic.co/search-ui/overview)
```
Comment (yakhinvadim) [inline README.md:38]:
```suggestion
Check out our [documentation site](https://docs.elastic.co/search-ui/overview).
```

File: README.md [hunk 2]
```diff
@@ -33,6 +33,10 @@ search experience up and running in minutes.
...
+
 ### Live Demo
 
 Checkout the [live demo of Search UI](https://search-ui-stable.netlify.com).
```
Comment (yakhinvadim) [inline README.md:42]:
```suggestion
Check out the [live demo of Search UI](https://search-ui-stable.netlify.com).
```

File: README.md [hunk 3]
```diff
@@ -155,7 +159,7 @@ const connector = new AppSearchAPIConnector({
 });
 ```
 
-_Read the [advanced README](./ADVANCED.md#build-your-own-connector) to learn how to build a connector for any Search API._
+_Read our [Custom Connector](https://docs.elastic.co/search-ui/connectors-api/custom-connector) to learn how to build a connector for any Search API._
```
Comment (yakhinvadim) [inline README.md:162]:
```suggestion
_Read our [Custom Connector Guide](https://docs.elastic.co/search-ui/connectors-api/custom-connector) to learn how to build a connector for any Search API._
```

**Review by yakhinvadim — DISMISSED** [review]
Just a couple small fixes :)

**Comment by elasticdocs** [comment]
:rocket: Built _docs-elastic-co_ successfully!

* [View deploy preview](https://docs-elastic-1zplhg9j6-elastic-dev.vercel.app)

* Built with commit c73b3e4ad3b329b4296fb285cd3378a4a49747fd

> Issues? Visit #next-docs in Slack

**Review by yakhinvadim — APPROVED** [review]

**Pull request merged**